statsmodels/stats/oneway.py

```diff
@@ -856,12 +1059,69 @@ def test_scale_oneway(data, method='bfm', center='median', transform='abs',
 
     Parameters
     ----------
-    data
-    method "equal", "unequal", "bfm"
+    data : tuple of array_like or DataFrame or Series
+        Data for k independent samples, with k >= 2.
+        The data can be provided as a tuple or list of arrays or in long
+        format with outcome observations in ``data`` and group membershipt in
+        ``groups``.
+    groups : ndarray or Series
+        If data is in long format, then groups is needed as indicator to which
+        group or sample and observations belongs.
+    method : {"unequal", "equal" or "bf"}
+        `use_var` specified how to treat heteroscedasticity, unequal variance,
+        across samples. Three approaches are available
+
+        "unequal" : Variances are not assumed to be equal across samples.
+            Heteroscedasticity is taken into account with Welch Anova and
+            Satterthwaite-Welch degrees of freedom.
+            This is the default.
+        "equal" : Variances are assumed to be equal across samples.
+            This is the standard Anova.
+        "bf: Variances are not assumed to be equal across samples.
+            The method is Browne-Forsythe (1971) for testing equality of means
+            with the corrected degrees of freedom by Merothra. The original BF
+            degrees of freedom are available as additional attributes in the
+            results instance, ``df_denom2`` and ``p_value2``.
+
     center : "median", "mean", "trimmed" or float
+        Statistic used for centering observations. Default is median.
     transform : "abs", "square" or callable
-    trim_frac_mean=0.1,
-    trim_frac_anova=0.
+        Transformation for the centered observations.
+        Default is absolute value.
+    trim_frac_mean=0.1 : float in [0, 0.5)
+        Trim fraction for the trimmed mean when `center` is "trimmed"
+    trim_frac_anova : float in [0, 0.5)
+        Optional trimming for Anova with trimmed mean and winsorized variances.
+        With the default trim_frac equal to zero, the oneway Anova statistics
+        are computed without trimming. If `trim_frac` is larger than zero,
+        then the largest and smallest observations in each sample are trimmed.
+        The number of trimmed observations is the fraction of number of
+        observations in the sample truncated to the next lower integer.
+        `trim_frac` has to be smaller than 0.5, however, if the fraction is
+        so large that there are not enough observations left over, then `nan`
+        will be returned.
+
+    Returns
+    -------
+    res : results instance
+        The returned HolderTuple instance has the following main attributes
+        and some additional information in other attributes.
+
+        statistic : float
```

Comment:
Also probably should use a list here rather than what will probably be a mangled definition list as is.

Reply:
Definition list actually looks good here.

![image](https://user-images.githubusercontent.com/5585221/88296022-15eb8880-ccf6-11ea-9f50-b24bcb606747.png)